table-binary-unicode-properties.html

```diff
@@ -12,22 +12,19 @@
       <td><a href="https://unicode.org/reports/tr18/#General_Category_Property">`ASCII`</a></td>
     </tr>
     <tr>
-      <td>
-        <ul>
-          <li>`ASCII_Hex_Digit`</li>
-          <li>`AHex`</li>
-        </ul>
+      <td>`ASCII_Hex_Digit`</td>
+      <td>`AHex`</td>
       </td>
```

Comment:
Seems stray?

Reply:
Figures, the only one I did by hand is messed up 🙄. Ugh.